fuzzers/064-gtp-channel-conf/generate.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+import json
+import os
+from enum import Enum
+
+from prjxray.segmaker import Segmaker
+
+INT = "INT"
+BIN = "BIN"
+BOOL = "BOOL"
+STR = "STR"
+
+
+def bitfilter(frame, bit):
+    # Filter out interconnect bits.
+    if frame not in [28, 29, 30, 31, 0, 1, 2, 3]:
+        return False
+
+    return True
```

Comment:
There should be two bitfilter functions, as the GTP_CHANNEL can have different frames based on the tile. See https://github.com/SymbiFlow/prjxray/pull/1548/commits/c59c486959b48ae5e449816f077f9e249c7ac45a.

Reply:
done